spec/requests/posts_spec.rb

```diff
@@ -1,17 +1,30 @@
 require 'rails_helper'
+require 'spec_helper'
 
 RSpec.describe 'Posts', type: :request do
-  describe 'GET /index' do
-    it 'returns http success' do
-      get '/posts/index'
-      expect(response).to have_http_status(:success)
+  before :each do
+    @post = Post.new(
+      title: 'Great Efforts yield Great Results',
+      text: 'Its always hard at first put with time you again',
+      comments_counter: 0,
+      likes_counter: 0
+    )
+    @post.save
+
+    describe 'GET /index' do
+      it 'renders the index template' do
+        get '/users/:id/posts/'
+        expect(response.status).to eq(200)
+        expect(response.body).to include('List all users posts')
+      end
     end
-  end
 
-  describe 'GET /show' do
-    it 'returns http success' do
-      get '/posts/show'
-      expect(response).to have_http_status(:success)
+    describe 'GET /show' do
+      it 'renders the show template' do
+        get "/users/posts/#{@post.id}"
+        expect(response.status).to eq(200)
+        expect(response).to render_template('show')
```

Comment:
- [ ] Great job creating the tests for `posts` requests however you should include tests that ensure the response body includes the correct placeholder text, you did this in the previous test but kindly make sure to include it also in this test. e.g `expect(response.body).to include(.....)`